scene/gui/code_edit.cpp

```diff
@@ -1655,6 +1655,11 @@ TypedArray<int> CodeEdit::get_folded_lines() const {
 	return folded_lines;
 }
 
+bool CodeEdit::is_line_hidden(int p_line) const {
+	ERR_FAIL_INDEX_V(p_line, text.size(), false);
```

Comment:
`text.size()` must be `get_line_count()` instead.

Reply:
Should I also update this one on the exposed method?

Reply:
I already updated just in case, because I'll offline for the next 12h